sdk/keyvault/tests.yml

```diff
@@ -5,17 +5,12 @@ extends:
   parameters:
     ServiceDirectory: keyvault
     TimeoutInMinutes: 180
+    AdditionalMatrixConfigs:
+      - Name: keyvault_test_matrix_addons
+        Path: sdk/keyvault/platform-matrix.json
```

Comment:
Should this be an absolute path like all the others?

Reply:
See https://github.com/Azure/azure-sdk-for-net/pull/19338#discussion_r596257495